### ISSUE TITLE
OACMPDM5 with different physics

### ISSUE THREAD

**Issue by The-Gig** [NeonKnightOA/oacmpvol1#12]
I'm checking old "todo lists" from the forum against z_oacmp1v4_beta-220322.pk3.

OACMPDM5: Jumping from the lateral areas of the upper floor to the platform in the middle of the arena is quite possible at 125 Hz physics, but gets hard moving to accurate and maybe impossible moving to 90 Hz physics. I don't know what was Jan's idea about that
![shot0233](https://user-images.githubusercontent.com/23258514/161793292-a597985f-118e-4e3f-9775-7e5583b1f893.jpg)
![shot0232](https://user-images.githubusercontent.com/23258514/161793295-729e80d7-97bb-4a48-a142-97d2d75c951d.jpg)
.

**Comment by NeonKnightOA** [comment]
Yeah, FPS-dependant jumps are a bad idea. I'll see what can I do.

**Comment by NeonKnightOA** [comment]
I'm trying a solution that should be put to test to see if it works.
